DAO.sol

```diff
@@ -500,8 +500,18 @@ contract DAO is DAOInterface, Token, TokenSale {
 
         uint quorum = p.yea + p.nay;
 
+        // require 53% for calling newContract()
+        bool updateContractQuorumCheck = true;
+        if (_transactionData.length >= 4 && _transactionData[0] == 0x68
+            && _transactionData[1] == 0x37 && _transactionData[2] == 0xff
+            && _transactionData[3] == 0x1e
```

Comment:
I'm trying to wrap my head around what these checks are doing. Are `0x68`, `0x37`, `0xf`f and `0x1e` some sort of opcodes? Where do I find their meaning?

This code might be easier to read if you put these hex numbers in a named `uint constant`.